### DISCUSSION TITLE
vi/models/yolov9/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#9090] [Docs]
# vi/models/yolov9/

Khám phá YOLOv9, sự bổ sung mới nhất cho kho vũ khí phát hiện đối tượng thời gian thực, tận dụng Thông tin Gradient có thể lập trình và kiến trúc GELAN cho hiệu suất vô song.

https://docs.ultralytics.com/vi/models/yolov9/

**Comment by nguyenvandang1998** [comment]
I saw your article about yolov9 and tried to load the yolov9 model with the command model = YOLO('yolov9c.pt') but it didn't work. error program "ERROR weights/yolov9-c-converted.pt appears to be an Ultralytics YOLOv5 model originally trained with https://github.com/ultralytics/yolov5. This model is NOT forwards compatible with YOLOv8 at https:// github.com/ultralytics/ultralytics. Recommend fixes are to train a new model using the latest 'ultralytics' package or to run a command with an official YOLOv8 model, i.e. 'yolo predict model=yolov8n.pt'. what should I do? How to load yolov9 model from ultralytics?
Thanks!

**Reply by glenn-jocher** [reply]
Hey there! 👋 It looks like you're trying to load a YOLOv9 model, which isn't directly compatible with our current setup. As YOLOv9 introduces some new architectures and training techniques, it needs a specific approach to be used.

To work with YOLOv9 in Ultralytics, you would need to ensure that you're using the right repository or package that explicitly supports YOLOv9 models. As of the moment, our Ultralytics repo focuses on versions up to YOLOv8. If YOLOv9 is developed by a separate open-source group as mentioned, it's best to check their specific implementation for loading and using YOLOv9 models. Make sure to follow their documentation and installation guides. 

For example, if they provide a Python package or a specific way to load models, ensure you're following their instructions closely. 

If you still want to work within the Ultralytics ecosystem and use the advancements we offer, training a new model with our latest package or using one of our pre-trained YOLOv8 models might be the best approach for now.

Hope this helps! If you have more questions, feel free to ask. 😊
